src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -43,10 +47,9 @@ public function new(Request $request): Response
     {
         $<?= $entity_var_singular ?> = new <?= $entity_class_name ?>();
         $form = $this->createForm(<?= $form_class_name ?>::class, $<?= $entity_var_singular ?>);
-        $form->handleRequest($request);
 
-        if ($form->isSubmitted() && $form->isValid()) {
-            $em = $this->getDoctrine()->getManager();
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
```

Comment:
We should revert this too. This change should be made to symfony-docs first, if we want it.